xenonhd.dependencies

```diff
@@ -1,2 +1,16 @@
 [
+  {
+    "repository": "TeamHorizon/android_device_asus_qcom_common",
+    "target_path": "device/asus/msm8916-common",
+    "branch": "n"
+  }
+  {
+    "repository": "LineageOS/android_kernel_asus_msm8916",
+    "target_path": "kernel/asus/msm8916",
+    "branch": "n"
+  }
+  {
+    "repository": "BlackFoxy757/proprietary_vendor_asus",
```

Comment:
this one should be forked as well. 
btw at the moment it's completely wrong as BlackFoxy757/proprietary_vendor_asus repo doesn't exist